sources/api/prairiedog/src/bootconfig.rs

```diff
@@ -94,23 +95,14 @@ where
 
 /// Retrieves boot config related Bottlerocket settings. If they don't exist in the settings model,
 /// we return `None` instead.
-async fn get_boot_config_settings<P>(socket_path: P) -> Result<Option<BootSettings>>
+async fn get_boot_config_settings<P>(config_path: P) -> Result<Option<BootSettings>>
 where
     P: AsRef<Path>,
 {
-    let uri = "/settings";
-    let settings: serde_json::Value =
-        schnauzer::v1::get_json(socket_path, uri, Some(("prefix", "boot")))
-            .await
-            .context(error::RetrieveSettingsSnafu)?;
```

Comment:
Mind removing the enum variants that aren't used anymore?

Reply:
sure thing